node/adapter/docker_restart.py

```diff
@@ -0,0 +1,125 @@
+import os
+import time
+import sh
+import fileinput
+import sys
+REAL_SOCK = "/var/run/docker.real.sock"
+POWERSTRIP_SOCK = "/var/run/docker.sock"
+
```

Comment:
This module would really benefit from a block comment explaining the Unix socket business and why we're going to all this trouble.